File: C_Language/Que_10.c
#include <stdio.h>
int main() {
    int arr[]={1,2,3,4,5};
    int cnt1=0;
    int cnt2=0;
    int len=sizeof(arr)/sizeof(arr[0]);
    int i;
    for(i=0;i<len;i++){
        if(arr[i]%2==0){
            cnt1+=1;
        }
        else{
            cnt2+=1;
        }
    }
    printf("no of odd no: %d\n",cnt2);
    printf("no of even no: %d",cnt1);
    return 0;
}

File: Java_Script/Que_10.js
// 10. Given an array of integers, count how many numbers are even and how many are odd.
// Examplef
// Input: [1, 2, 3, 4, 5]
// Output: Even: 2, Odd: 3
let arr=[1,2,3,4,5];
let i,cnt1=0,cnt2=0;
for(i=0;i<arr.length;i++){
    if(arr[i]%2==0){
        cnt1+=1;
    }
    else{
        cnt2+=1;
    }
}
console.log("even nos:",cnt1);
console.log("odd nos:",cnt2);